frontend/src/domain/reference.ts

```diff
@@ -50,9 +50,11 @@ export default class Reference implements EditableListItem {
 
   get renderDecision(): string {
     return [
+      this.author ? `${this.author},` : "",
       this.legalPeriodical?.abbreviation ?? this.legalPeriodicalRawValue,
       this.citation,
       this.referenceSupplement ? ` (${this.referenceSupplement})` : "",
+      this.documentType ? ` (${this.documentType.jurisShortcut})` : "",
```

Comment:
@leonie-koch For some reason, this does not work for saved decisions. Looks a bit like the objects returned by the servie are not of the right type (at least I had similar problems with that)

Reply:
Same with the Author

Reply:
Ah, I think I finally understood the problem. In the backend, we need to save literatureReferences separately from the references. Right now, they are saved as normal references and therefore the author and documentType are always `null`